scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+import subprocess
+import os
+import sys
+import getopt
+import json
+import time
+from socket import gethostname
+from typing import Dict, List, NamedTuple, Set
+import re
+from datetime import datetime
+from collections import OrderedDict
+lnet_info = {}
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+from cortx.utils.schema.payload import Json
+from cortx.utils.conf_store import ConfStore
+
+LNET_CONF_FILE = "/etc/modprobe.d/lnet.conf"
+TIMEOUT = 180
+
+class motr_prov:
+    def __init__(self, index, url):
+        self.conf_store = ConfStore()
+        self.url = url
+        self.index = index
+        self.load_config(self.index, self.url)
+        self.server_id = int(self.conf_store.get(self.index, 'cluster>current>server_id', default_val=None))
+
+    def configure_lnet_from_conf_store(self):
+        '''
+           Get iface and /etc/modprobe.d/lnet.conf params from
+           conf store. Configure lnet. Start lnet service
+        '''
+        lnet_conf = self.conf_store.get(self.index, f'cluster>server[{self.server_id}]>network>motr_net', default_val=None)
+        with open(LNET_CONF_FILE, "w") as fp:
+            fp.write(f"options lnet networks={lnet_conf['interface_type']}({lnet_conf['interface']})  config_on_load=1  lnet_peer_discovery_disabled=1\n")
+        time.sleep(10)
+        self.start_services(["lnet"])
+
+    def execute_command(self, cmd, timeout_secs):
+        
+        ps = subprocess.Popen(cmd, stdin=subprocess.PIPE,
```

Comment:
Please use SimpleProcess() a wrapper to simply the calls 
https://github.com/Seagate/cortx-utils/blob/main/py-utils/src/utils/process.py
Usage Example: https://github.com/Seagate/cortx-utils/blob/main/py-utils/src/utils/validator/v_network.py

Reply:
Because of few limitations, SimpleProcess() review comment is not addressed now.
Created issue about SimpleProcess:
https://github.com/Seagate/cortx-utils/issues/151